src/vis_cpu/conversions.py

```diff
@@ -1,24 +1,185 @@
 """Functions for converting co-ordinates."""
+import astropy.units as u
 import numpy as np
+from astropy.coordinates import EarthLocation, SkyCoord
+from astropy.coordinates.builtin_frames import AltAz
+from astropy.time import Time
 
 
-def lm_to_az_za(el, m):
-    """
-    Convert l and m (on intervals -1, +1) to azimuth and zenith angle.
+def enu_to_az_za(enu_e, enu_n):
+    """Convert angle cosines in ENU coordinates into azimuth and zenith angle.
+
+    For a pointing vector in ENU coordinates vec{p}, the input arguments are
+    ``enu_e = vec{p}.hat{e}`` and ``enu_n = vec{p}.hat{n}`, where ``hat{e}`` is
+    a unit vector in ENU coordinates etc.
```

Comment:
Can we note that these `enu_e/n` are the same as `el, m` for a standard co-ordinate frame?

Reply:
Yes, done. There was some confusion to do with the orientation of the azimuth coordinate, but hopefully that is all treated very explicitly now.